TeachEasy/src/teacheasy/data/ButtonData.java

```diff
@@ -0,0 +1,85 @@
+package teacheasy.data;
+/**
+ * @version 1.0 05 Feb 2015
+ * @author sh1157 and sr896
+ */
+
+/** Class to hold Button Data */
+public class ButtonData extends PageObject {
+	

```

Comment:
Style: Potentially unnecessary white space. Should be replaced by a Documentation comment for the enumerated type